src/pages/AboutAppsPage/AboutAppsPage.jsx

```diff
@@ -149,12 +151,39 @@ export default function AboutAppsPage() {
 
   return (
     <S.Root>
+      <Header />
       <S.TopToIntroContainer>
         <S.Top>
           <S.PageTitleWrapper>
             <S.PageTitle>ABOUT</S.PageTitle>
+            <S.PageSubTitle>APPS 소개</S.PageSubTitle>
             <S.IconTitle></S.IconTitle>
           </S.PageTitleWrapper>
+          <S.SquareContainer>
+            {STYLED_SQUARE_COLOR_CARDS.map((colors) => {
+              return (
+                <S.StyledSquareWrapper>
+                  {colors.map((left) => {
+                    return (
+                      <S.StyledSquares>
+                        {left.map((c) => {
+                          return (
+                            <>
+                              {c === 'square2' ? (
+                                <S.StyledSquare2></S.StyledSquare2>
+                              ) : (
+                                <S.StyledSquare color={c}></S.StyledSquare>
+                              )}
+                            </>
+                          );
+                        })}
+                      </S.StyledSquares>
+                    );
+                  })}
+                </S.StyledSquareWrapper>
+              );
+            })}
+          </S.SquareContainer>
```

Comment:
```suggestion
          <S.SquareContainer>
  {STYLED_SQUARE_COLOR_CARDS.map((colors) => (
    <S.StyledSquareWrapper>
      {colors.map((left) => (
        <S.StyledSquares>
          {left.map((c) => (
            c === 'square2' ? (
              <S.StyledSquare2></S.StyledSquare2>
            ) : (
              <S.StyledSquare color={c}></S.StyledSquare>
            )
          ))}
        </S.StyledSquares>
      ))}
    </S.StyledSquareWrapper>
  ))}
</S.SquareContainer>
```

return문이 반복되어 가독성이 떨어지는데, arrow function 형식으로 수정해보면 어떨까요?